src/main/kotlin/com/ondosee/domain/notification/presentation/NotificationController.kt

```diff
@@ -1,20 +1,21 @@
 package com.ondosee.domain.notification.presentation
 
+import com.ondosee.domain.notification.presentation.web.req.SetAlarmWebRequest
 import com.ondosee.domain.notification.service.NotificationService
 import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestMapping
-import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
+import javax.validation.Valid
 
 @RestController
 @RequestMapping("/notification")
 class NotificationController(
     private val notificationService: NotificationService
 ) {
     @PostMapping
-    fun saveNotification(@RequestParam deviceToken: String): ResponseEntity<Void> =
-        notificationService.saveNotification(deviceToken)
+    fun setAlarm(@Valid webRequest: SetAlarmWebRequest): ResponseEntity<Void> =
```

Comment:
```suggestion
    fun setAlarm(@Valid webRequest: SetAlarmWebRequest): ResponseEntity<Unit> =
```
Kotlin의 Unit을 사용하는 건 어떨까요?

Reply:
[c5019a6](https://github.com/ondosee/ondosee-Backend/pull/17/commits/c5019a6ffcc4bb68f69096142f8e6ce76064d56b)

수정했습니다!!